frontend/src/FindQuiz.js

```diff
@@ -45,15 +50,20 @@ class FindQuizContainer extends React.Component {
         let restaurant = this.state.Restaurants.pop();
         this.setState({ Restaurants: [] });
         this.setState({ showRes: false });
-        this.rateRestaurant(true, restaurant.id);
+        this.rateRestaurant(1, restaurant.id);
         alert("Have a nice meal! After you eat, don't forget to rate your experience for even better recommendations!");
     }
-
+    later = (e) =>{
+        e.preventDefault()
+        let restaurant = this.state.Restaurants.pop();
+        this.handleSubmit(e);   // Get new restaurant
+        this.rateRestaurant(0, restaurant.id);
+    }
     reject = (e) => {
         e.preventDefault()
         let restaurant = this.state.Restaurants.pop();
         this.handleSubmit(e);   // Get new restaurant
-        this.rateRestaurant(false, restaurant.id);
+        this.rateRestaurant(-1, restaurant.id);
```

Comment:
This is connected to my comment above. The this.rateRestaurant calls are currently sending a value "1", "0", or "-1" to app.py, which does not match the conditionals. One of the two files needs to be changed so that the conditionals work as expected and reviews are populated in the review tab.

Reply:
Changes implemented. FindQuiz now returns the values the backend expects.